packages/vaadin-icon/src/vaadin-iconset.d.ts

```diff
@@ -15,11 +15,16 @@ declare class IconsetElement extends ElementMixin(HTMLElement) {
    */
   name: string;
 
+  /**
+   * The size of an individual icon. Note that icons must be square.
+   */
+  size: number;
+
   /**
    * Produce SVGTemplateResult for the element matching `id` in this
    * iconset, or `undefined` if there is no matching element.
    */
-  applyIcon(name: string): IconSvgLiteral;
+  applyIcon(name: string): { svg: IconSvgLiteral; size: number };
```

Comment:
When looking at the tests below I see that the icon id here may or may not inclue the iconset name as a prefix. Should it be documented that the parameter supports both options?

Reply:
Added docs.